### PR TITLE
Update installation instructions to use miniforge/mamba

### PR DESCRIPTION
**Pull request by ktyle**
Addresses #391

### CODE REVIEW

**Comment by ktyle** [comment]
A couple of links to NOAA NCEI are failing. These are due to impacts from Hurricane Helene and should eventually work again.

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Ignoring the expected link check failures in order to finally get this in.

File: foundations/conda.md [hunk 1]
```diff
@@ -31,16 +31,17 @@ Package management is useful because you may want to update a package for one of
...
+Miniforge uses the `conda` package management system and is based on Miniconda, which is a pared-down version of the full Anaconda Python distribution.
 
 [Installing Anaconda](https://docs.anaconda.com/anaconda/install/) takes longer and uses up more disk space, but provides you with more functionality, including Spyder (a Python-specific integrated development environment or IDE) and Jupyter, in addition to other immediately installed packages. Also, the interface of Anaconda is great if you are uncomfortable with the terminal.
 
-We recommend Miniconda for two reasons:
+We recommend Miniforge for these reasons:
 
 1. It's quicker and takes up less disk space.
 2. It encourages you to install only the packages you need in reproducible isolated environments for specific projects. This is generally a more robust way to work with open source tools.
+3. It uses `conda-forge` as the default channel for packages.
```
Comment (brian-rose) [inline foundations/conda.md:44]: This is great, but might not mean much to the newbie. Here's my suggested addition:
```suggestion
3. It uses `conda-forge` as the default channel, which is our recommended way to get up-to-date, interoperable packages.
```

**Review by brian-rose — COMMENTED** [review]
Looks great @ktyle, just one small recommended addition.

File: foundations/conda.md (hunk 1 above)
Comment (dopplershift) [reply foundations/conda.md:44]: Oops, looks like I clicked the merge button too quickly.

Comment (ktyle) [reply foundations/conda.md:44]: no prob, i'll commit and merge the suggestion by @brian-rose.

**Pull request merged**